cmd/snap-bootstrap/degraded-recover-mode.svg

```diff
@@ -0,0 +1,3 @@
eignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 840px; margin-left: 242px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 11px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">done</div></div></div></foreignObject><text x="242" y="843" fill="#000000" font-family="Helvetica" font-size="11px" text-anchor="middle">done</text></switch></g><rect x="187" y="710" width="120" height="60" rx="9" ry="9" fill="#ffffff" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 740px; margin-left: 188px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">mount save (s8)</div></div></div></foreignObject><text x="247" y="744" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">mount save (s8)</text></switch></g><rect x="47" y="810" width="120" height="60" rx="9" ry="9" fill="#ffffff" stroke="#000000" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 840px; margin-left: 48px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">done (s9)</div></div></div></foreignObject><text x="107" y="844" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">done (s9)</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
I think it'd tweak the diagram so that the done (final) state is at the bottom. Then if we group the fallback states on one side there should be no need for transition lines to go across the whole diagram. I can look into reorganizing this in a follow up.

Reply:
the label "fail or success, unencrypted" reads a bit ambiguously. Maybe in general instead of using ","  we should use parentheses, so

`fail or success(unencrypted)`

perhaps?

Reply:
I can't seem to fix the layout as you described, it's a bit difficult because so many states can go to the `unlock save with fallback key` state, but I did revise the labels on the directed edges as @pedronis recommended